ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ip-nat.yml

```diff
@@ -3,30 +3,33 @@ ip_nat:
   kernel_buffer_size: 64
   pools:
     - name: prefix_16
+      type: port-only
       prefix_length: 16
       ranges:
-        - first_ip: 10.0.0.1
-          last_ip: 10.0.255.254
+        - first_port: 1023
+          last_port: 65534
         - first_ip: 10.1.0.0
```

Comment:
Keep this prefix_16 test as it was and create another one if `port-only` and add a comment stating that `first_ip` / `last_ip` will be ignored!

Reply:
Done